tests/integration/it_floors_test.py

```diff
@@ -16,6 +15,32 @@ def test_list(client, db_session):
     assert b"Test floor" in floors.data
 
 
+@pytest.mark.skip(reason="Order is not yet implemented")
+def test_ordered_list(client):
+    factories.FloorFactory(description="B")
+    factories.FloorFactory(description="A")
+    response = client.get(url_for("floor.list", order_by=["description:asc"]))
+    html = response.data.decode('utf-8')
+    assert html.count("""</header>
+
+
+                    <article class=\"floor\">
+                    <header>
+                        <div>
+                        <h1>A</h1>
+    """) == 1
```

Comment:
@emilianodellacasa indentation here seems strange

Reply:
@amihaiemil I am not sure yet if this is the right way to test it, but I fixed the identation